packages/theme/src/slider-button/vars.less

```
@@ -0,0 +1,14 @@
+.inject-SliderButton-vars() {
+  // 次要文本色
+  --tv-Slider-Button-color-text-secondary: var(--tv-color-text-secondary, #595959);
+  // 品牌色 / 选中色
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)